api.go

```diff
@@ -214,6 +217,10 @@ type (
 		ProjectID  string      `json:"projectId,omitempty"`
 	}
 
+	RecordingSpec struct {
```

Comment:
Do we want maybe to move recording check to MediaGuard?

CC: @thomshutt @pwilczynskiclearcode

Reply:
I think that Recording is something I'd rather have the E2Es cover, since it's a feature rather than a fundamental indicator of core workflow / system health

Reply:
Cool. My plan was to add it to E2E for now while MediaGuard doesn't handle recordings anyway. But makes sense to keep it only there on the long-run.